.github/ISSUE_TEMPLATE/release-patch.md

```diff
@@ -18,6 +18,8 @@ Pre-release
   - [ ] Update semantic version in `CMakeLists.txt`
   - [ ] Update changelog below and agree on it with everyone; then commit it to `docs/changelog` in the cherrypicks PR (copy structure as-is)
   - [ ] Update `docs/index.rst` accordingly with the new `.pkg` and `.exe` links for `PKG installer` and `Windows Installer`
+- [ ] Activate ReadtheDocs for the cherry-pick branch and ensure the documentation builds
```

Comment:
possible to add a link to the right page on readthedocs?

Reply:
Most certainly!  
A link and a small explanation.